vllm/engine/llm_engine.py

```diff
@@ -316,14 +373,24 @@ def __del__(self):
         if model_executor := getattr(self, "model_executor", None):
             model_executor.shutdown()
 
+    MISSING_TOKENIZER_GROUP_MSG = ("Unable to get tokenizer because "
+                                   "skip_tokenizer_init is True")
+
+    def get_tokenizer_group(self, fail_msg: str = MISSING_TOKENIZER_GROUP_MSG):
```

Comment:
Could you add return type hint

Why is the error message an arg here?

Reply:
> Could you add return type hint

Done

> Why is the error message an arg here?

It was suggested by @ywang96 earlier

Reply:
The rationale is that we might fail to get the tokenizer due to different reasons at different places, but I'm okay with making it not an arg for now.

Reply:
Since it's already done, I might as well keep it.

Reply:
No problem can always rework this in a later PR if we decide that.